docs/conf/eu/2017/cfp.rst

```diff
@@ -3,5 +3,113 @@
 Call for proposals
 ==================
 
-We're still ironing out the call for proposals for our North America conference in Portland,
-stay tuned for updates to this page as we get ready to launch the CFP for Europe!
+Hello hello, fellow documentarians! It's that time of year again: We’re very
+excited to announce that we are now accepting talk proposals for our next North
+American conference, coming up on **September 10-12, 2017, in Prague, Czech Republic**.
+
+Every year, Write the Docs invites people from all across our community to come
+up on stage to share their insights and experience. Whether you’ve been writing
+documentation for decades or you’ve just started this year, we want to hear from
+you! Read on to learn more about the goals of the conference and what we look
+for in talk proposals.
+
+In the meantime, mark your calendars: **The deadline for submitting proposals is
+Midnight PST on May 31st, 2017.** We aim to let you know if your proposal is
+accepted by 15th June.
+
+Conference goals
```

Comment:
I'm assuming this is the same copy for NA, but EUified. Will skim but I doubt we need to make major changes here ;)